ord_schema/validations.py

```diff
@@ -503,3 +606,11 @@ def validate_binary_data(message):
     reaction_pb2.Percentage: validate_percentage,
     reaction_pb2.BinaryData: validate_binary_data,
 }
+
+COMPOUND_STRUCTURAL_IDENTIFIERS_ = [
```

Comment:
A couple style comments:
* Please put constants at the top of the module, where possible (the validators dict is an exception since it depends on functions being defined above it).
* Indicate _PRIVATE constants with a leading (not trailing) underscore; see [here](http://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations).

Reply:
Done